### PR TITLE
added error message for 500 to logger

### PR DESCRIPTION
**Pull request by ywkim312**
## Description
Added error messages for 500 error to event service. When 500 error happens, the logger will have a brief message about the reason of the error.

**Fixes #841**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Comment by bingzhang** [comment]
@ywkim312 @sandeep-ps I can not leave comments on the code line. I am not sure why.

**Comment by bingzhang** [comment]
`import configs.messages as msgs` might change to `import controllers.messages as msgs`

**Comment by ywkim312** [comment]
> `import configs.messages as msgs` might change to `import controllers.messages as msgs`

changed.

File: eventservice/api/controllers/events.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
 from time import gmtime
 
 import controllers.configs as cfg
+import controllers.configs.messages as msgs
```
Comment (bingzhang) [inline eventservice/api/controllers/events.py:33]: it works.

**Review by bingzhang — APPROVED** [review]

**Comment by ywkim312** [comment]
main.py's line 44 makes an error based on situation. Maybe you'd better make it as separate variable in conf instead of hard code it.

**Comment by bingzhang** [comment]
> main.py's line 44 makes an error based on situation. Maybe you'd better make it as separate variable in conf instead of hard code it.

@ywkim312 are you leaving the comments at the different pr?

**Comment by ywkim312** [comment]
> main.py's line 44 makes an error based on situation. Maybe you'd better make it as separate variable in conf instead of hard code it.

oops sorry. thank you for catching it. =)

**Review by sandeep-ps — APPROVED** [review]

**Comment by sandeep-ps** [comment]
@bingzhang, I'm going to merge this as you had approved it earlier. Please let me know if there are any concerns. Thanks.

**Pull request merged**